addons/crm/models/crm_lead.py

```diff
@@ -98,6 +98,7 @@ class Lead(models.Model):
                 'mail.activity.mixin',
                 'utm.mixin',
                 'format.address.mixin',
+                'statusbar.time.mixin',
```

Comment:
Side note: would do 1 commit for the mixin implementation, then 1 commit that adds it into the relevant models, just to split a bit things. Moreover it allows to split the explanation on the mixin (what, how, ...) from why we need it on some specific models (functional explanation).